.eslintrc.json

```diff
@@ -12,6 +12,7 @@
     },
     "globals": {
         "_": true,
+        "__WEBPACK_REGION__": true,
```

Comment:
Since it's used only once, you can disable it by using inline comments.

Reply:
I do not really like the inline style (or only for TODO), simply because if one day we don't use eslint anymore, I'd just like to delete the eslint file and not have to grep all the code to remove inline comments.